README.md

```diff
@@ -176,95 +176,138 @@ After the loop completes, have the function return the original array.
 
 ## The `while` loop
 
-The `while` loop is similar to a `for` loop in that it will keep looping as long
-as the condition evaluates to `true`. However, the idea behind a `for` loop is
-that it should loop a set number of times (e.g., start with `i = 0` and loop
-while `i < 25`). A `while` loop, on the other hand, is designed to loop an
-indefinite amount of times — it should keep looping **while** the condition is
-`true` or, said differently, **until** the condition is `false`. There isn't a
-definite limit like "25 times" placed upon it.
-
-A `while` loop has the following structure:
+The `while` loop is similar to a `for` loop, repeating an action in a loop based
+on a condition. Both will continue to loop until that condition evaluates to
+`false`. Unlike `for`, `while` only requires condition and loop statements:
 
 ```js
 while ([condition]) {
   [loop body]
 }
 ```
 
-**_Usage_**: A `while` loop is best used when we don't know how many times a loop needs to
-run. Often, this is because the condition depends on a dynamic value.
-
-#### Example
+The initialization and iteration statements of the `for` loop have not
+disappeared, though. In fact, we could rewrite our original `for` loop gift
+wrapping example using a `while` loop and achieve the exact same result:
 
 ```js
-function maybeTrue() {
-	return Math.random() >= 0.5;
-}
+const gifts = ["teddy bear", "drone", "doll"];
 
-while (maybeTrue()) {
-	console.log('And I ran... I ran so far away!');
+function wrapGifts(gifts) {
+  let i = 0; // the initialization!
```

Comment:
i'd all "initialization..***OUTSIDE*** the body of the loop.

n00bs toast that all the time.